setup_scripts/install_alglib.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# Note: This file should be removed once alglib is available through rosdep
+
+
+echo "================================================================"
+echo "installing Alglib library"
+echo "================================================================"
+
+# update the package index
+sudo apt-get update
+
+# install 
+sudo apt-get install libalglib-dev
```

Comment:
My bad for not seeing this PR before merge, but would this without `-y` prevent script from continuing on its own?

Reply:
Looking at the Travis CI log, it seems that `libalglib-dev` was already installed at [line 5740](https://travis-ci.org/github/UBC-Snowbots/Snowflake/builds/743111287#L5740) before this script began running at [line 7639](https://travis-ci.org/github/UBC-Snowbots/Snowflake/builds/743111287#L7639). As a result, when this command was executed, it didn't do anything. That said I remember having to install `libalglib-dev` manually while building locally since `rosdep` didn't auto install `libalglib-dev`. @shinhs0506 do you know how the other scripts managed to install `libalglib-dev` before this was executed?

Reply:
@raadkhan @ihsan314 `libalglib-dev` is now available through `rosdep`, I will remove this script and make another pull request